test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Tests/DataServicesClient/DefaultODataBatchHandlerTests.cs

```diff
@@ -311,19 +324,29 @@ protected static IEdmModel GetEdmModel(ODataModelBuilder builder)
             using (var messageReader = new ODataMessageReader(odataResponseMessage, new ODataMessageReaderSettings(), GetEdmModel(new ODataConventionModelBuilder())))
             {
                 var batchReader = messageReader.CreateODataBatchReader();
+
                 while (batchReader.Read())
                 {
                     switch (batchReader.State)
                     {
                         case ODataBatchReaderState.Operation:
                             var operationMessage = batchReader.CreateOperationResponseMessage();
                             subResponseCount++;
-                            Assert.Equal(201, operationMessage.StatusCode);
+
+                            if (operationMessage.ContentId.Equals("0"))
+                            {   // No-Content response for PATCH
```

Comment:
nit: does stylecop complain if the comment is in the same line as '{' ?

Reply:
Fixed.
I am not seeing stylecop being ran in the build on my branch. Anyway, it is good to keep it consistent.

---
In reply to: [204492201](https://github.com/OData/WebApi/pull/1545#discussion_r204492201) [](ancestors = 204492201)